rollbar/logger.py

```diff
@@ -68,6 +66,9 @@ def setHistoryLevel(self, level):
         logging.Handler.setLevel(self, level)
 
     def emit(self, record):
+        if record.name == 'rollbar':
```

Comment:
For good measure, how about if we also hardcode the name of the logger in `rollbar/__init__py` to be the string `"rollbar"`? (Will make it more greppable.)